.travis.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         directories:
           - $BUILD_DIR
       script:
-        - ${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories SVNDataLong,SVNData,RecoveryTool,ProblematicPath
+        - ${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories SVNDataLong,SVNData,RecoveryTool,ProblematicPath,SymLink
```

Comment:
Can we add the `SymLink` category to `pipeline/start.sh`?  This file is used to run the Docker-based tests locally.

Reply:
Will do.  I see that the `ProblematicPath` category does not appear in `pipeline/start.sh`.  Should it?  (I'm not familiar with any of this -- I was just trying to follow patterns.)

Reply:
`ProblematicPath` is added in pull request #4279.

Reply:
I just merged #4279 so that we can avoid potential merge conflicts later.

Reply:
Sounds good, thanks!

Reply:
Addressed in latest commit.